wagtail_wordpress_import/test/tests/test_wordpress_item.py

```diff
@@ -155,12 +155,12 @@ def setUp(self):
 
     def test_category_snippets_are_saved(self):
         snippets = Category.objects.all()
-        self.assertEqual(len(snippets), 4)
+        self.assertEqual(len(snippets), 5)
 
     def test_page_one_has_categories(self):
         page_one = self.imported_pages.get(title="Item one title")
         categories = page_one.specific.categories.all()
-        self.assertEqual(2, categories.count())
+        self.assertEqual(3, categories.count())
         self.assertEqual(categories[0].name, "Blogging")
         self.assertEqual(categories[1].name, "Life")
```

Comment:
**Suggestion**: Add a test that `categories[2].name == "2016"`

Reply:
I added the test 👍